modules/bundle/custom_component/README.md

```diff
@@ -0,0 +1,15 @@
+# Description
+This example mainly shows a typical use case that brings customized python component(like: transform, network, metrics, etc.) in the `train` config.
+
+Please note that this example depends on the `spleen_segmentation` bundle example and executes via overriding the config file of it.
+
+## commands example
+Export the customized python code to `PYTHONPATH`:
```

Comment:
perhaps comments that "To run the workflow with customized components, `PYTHONPATH` should be revised to include the path to the customized component."

and we should make sure the folder `custom_component/scripts` is a valid python module (it has a `__init__.py` file in the folder)

Reply:
updated in the latest commit.

Thanks.